### PR TITLE
Add process to duplicate events

### PR DESCRIPTION
**Pull request by guiomar**
Hi!

I have tried to do a `process_evt_duplicate` funtion, related to issue #536.
Could something like this work? It seems to work on my side.
(I didn't know how to set the new color)

Thanks!

### CODE REVIEW

File: toolbox/process/functions/process_evt_duplicate.m [hunk 1]
```diff
@@ -0,0 +1,148 @@
+function varargout = process_evt_duplicate( varargin )
+% PROCESS_EVT_DELETE: Delete a list of events.
```
Comment (juangpc) [inline toolbox/process/functions/process_evt_duplicate.m:2]: Shouldn't this be: "ROCESS_EVT_DUPLICATE: Duplicate a list of events. ?

**Comment by juangpc** [comment]
Hi Guio, ✋🏻 
Long time no see. Hope you are doing well.

**Comment by guiomar** [comment]
Hola Juan!! Muchos besos :)

File: toolbox/process/functions/process_evt_duplicate.m (hunk 1 above)
Comment (guiomar) [reply toolbox/process/functions/process_evt_duplicate.m]: Chenged now :)

**Comment by ftadel** [comment]
Hi Guio,

I would like to propose an alternative to adding yet another process, while we can adjust the existing ones to do what we want.
The number of processes in Brainstorm is slowly becoming an annoying bottleneck: it takes more and more time to parse the process folders and to create the process menus in the Pipeline editor. On older computers (including my laptop), it's already becoming quite annoying. Therefore, whenever we can with little loss of usability, I would prefer to add features without adding process functions/menus.

I propose this change to transform the process "Merge events" into "Duplicate / merge events", as these are really in essence the same operations: https://github.com/brainstorm-tools/brainstorm3/commit/0e87b06fdc97278f878e8eafaebe0a5625021d98

image

I'll close the PR and issue for now.
Please feel free to reopen it if you think there is something that is not handled correctly with my commit.

**Comment by guiomar** [comment]
Yes! Sounds very good! I didn't know this trick that @rcassani explained using the merge function :)
Clarifying it in the name will work nicely. For the duplicate funtion do you have to write the event twice above, or has it been modified to work just with one label? 
Just discard my PR. Thanks a lot!!

**Comment by ftadel** [comment]
> For the duplicate funtion do you have to write the event twice above, or has it been modified to work just with one label?

I removed the test that was forcing the process to have two input events.
So you should be able to make it work just like on the screen capture I posted above.

**Comment by guiomar** [comment]
Awesome!

**Pull request closed without merging**